XarmRTConnection:
  ros__parameters:
    robot_ip: "192.168.88.200"
    dof: 7
    joint_names: [ 'joint1', 'joint2', 'joint3', 'joint4', 'joint5', 'joint6', 'joint7' ]